index.html

```diff
@@ -1,49 +1,296 @@
-                                alt="twitter-Icon"></a></li>
-                    <li><a href="www.Linkdedin.com"> <img src="./Assets/images/smm-icons/Linkedin-Icon.svg"
-                                alt="Linkedin-Icon"></a></li>
-                    <li><a href="#"> <img src="./Assets/images/smm-icons/M-Icon.svg" alt="M-Icon"></a></li>
-                    <li><a href="www.github.com"> <img src="./Assets/images/smm-icons/github-Icon.svg"
-                                alt="github-Icon"></a></li>
-                    <li><a href="#"> <img src="./Assets/images/smm-icons/victory-Icon.svg" alt="victory-Icon"></a></li>
-                </ul>
-            </div>
+      <div class="intro">
+        <h1 class="title">
+          I’m Nabeel <br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="connectTitle">LET’S CONNECT</p>
+        <div class="connectBox">
+          <ul class="smm-icons">
+            <li>
+              <a href="www.twitter.com">
+                <img
+                  src="./Assets/images/smm-icons/twitter-Icon.svg"
+                  alt="twitter-Icon"
+              /></a>
+            </li>
+            <li>
+              <a href="www.Linkdedin.com">
+                <img
+                  src="./Assets/images/smm-icons/Linkedin-Icon.svg"
+                  alt="Linkedin-Icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./Assets/images/smm-icons/M-Icon.svg" alt="M-Icon"
+              /></a>
+            </li>
+            <li>
+              <a href="www.github.com">
+                <img
+                  src="./Assets/images/smm-icons/github-Icon.svg"
+                  alt="github-Icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#">
+                <img
+                  src="./Assets/images/smm-icons/victory-Icon.svg"
+                  alt="victory-Icon"
+              /></a>
+            </li>
+          </ul>
         </div>
+      </div>
     </section>
-</body>
-
-</html>
\ No newline at end of file
+    <section class="worksCards">
+      <div class="card" id="card1">
+        <div class="portfolioPic portfolioPic1"></div>
+        <div class="workDescription">
+          <div class="project">
+            <h3 class="projectTitle">Tonic</h3>
+            <ul class="clientDetail">
+              <li class="clientName">CANOPY</li>
+              <li><img src="./Assets/images/dot.svg" alt="dot" /></li>
+              <li>Back End Dev</li>
+              <li><img src="./Assets/images/dot.svg" alt="dot" /></li>
+              <li>2015</li>
+            </ul>
+          </div>
```

Comment:
- [ ] The project cards appear as seen in the image below when I view your project using the Chrome browser. Whereas, when I view it with Mozilla firefox it appears perfectly. I cannot ascertain if it is from my end or an issue with the project. Kindly check from your end and be sure it is not an issue that is within your style declarations. 

![image](https://user-images.githubusercontent.com/84156714/219682222-5b76212c-81f0-49cd-b471-d895d40b4bd0.png)

**_Notice how the button overlaps the container._**

Reply:
Hi @bobb-Rob,

Thank you so much for the nice feedback. I shall make these changes in a while.  

Regards